### PR TITLE
Clarify that the ACS is a minimal information model

### LINKED ISSUE #349: Clarify that the ACS information model is normative
We're doing a lot of work to translate the behavior of triples into the information they convey when verified. We need to ensure that verifier implementers understand that they need to present this information in their chosen data model to later stages.

### PR DESCRIPTION
**Pull request by deeglaze**
This makes the appraisal specification of every triple normative for what later phases can depend on.
Fixes #349

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -1504,7 +1504,7 @@ Non-CoRIM-based data structures require mapping transformation, but these are ou
 If a CoRIM profile is specified, there are a few well-defined points in the procedure where Verifier behaviour depends on the profile.
 The CoRIM profile MUST provide a description of the expected Verifier behavior for each of those well-defined points.
 
-Verifier implementations MUST exhibit the same externally visible behavior as described in this specification.
+Verifier implementations MUST provide the specified information model of the ACS at minimum following phase 4 as described in this specification.
```
Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:1507]:
```suggestion
Verifier implementations MUST provide the specified information model of the ACS at minimum following first four phases as described in this specification.
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Review by henkbirkholz — APPROVED** [review]

**Review by thomas-fossati — APPROVED** [review]
I still think the wording is awkward, but at least we all seem to agree on what this means ;-)

**Review by nedmsmith — APPROVED** [review]

**Review by henkbirkholz — APPROVED** [review]

**Comment by deeglaze** [comment]
I don't understand the build failure `Makefile:25: *** target file 'draft-ietf-rats-corim.xml' has both : and :: entries.  Stop.`

**Comment by nedmsmith** [comment]
> I don't understand the build failure `Makefile:25: *** target file 'draft-ietf-rats-corim.xml' has both : and :: entries. Stop.`

I fixed it in top level Makefile

**Pull request merged**